### PR TITLE
Reconceptualize via burning it to the ground

### PR DESCRIPTION
**Pull request by markrawls**
I present to you: The only stuff we actually care about for building new apps. We're moving to vanilla MUI for the most part.

Things I will be working out with Ayush when he has time and will be in a later PR:
- Theming of our tables
- Whether or not Averta is gonna stay (currently "maybe")
- Precise specs of typography, particularly in regards to buttons

I would advise browsing by commit instead of by diff as I completely regenerated the project for this one.

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    },
+  },
+});
+
+// Add in breakpoint helpers by device type. It is easier
+//   to figure out what is going on when using these names.
+theme.breakpoints.phone = theme.breakpoints.down('xs');
+theme.breakpoints.tablet = theme.breakpoints.down('sm');
+theme.breakpoints.notPhone = theme.breakpoints.up('sm');
+theme.breakpoints.desktop = theme.breakpoints.up('md');
```
Comment (Ian-MacLeod) [inline src/theme.js:87]: We can keep these, but I think using the breakpoints explicitly is actually clearer to me.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -12,5 +12,7 @@ jobs:
           node-version: '15.x'
       - name: Install dependencies
         run: yarn install
+      - name: Install peer dependencies
+        run: yarn add --peer --pure-lockfile @material-ui/core react react-dom
```
Comment (Ian-MacLeod) [inline .github/workflows/lint.yml:16]: I'm not sure I understand why this is necessary - wouldn't these be covered by the yarn install?

File: src/theme.js (hunk 1 above)
Comment (markrawls) [reply src/theme.js:87]: I agree with you actually. Nuking em.

File: .github/workflows/lint.yml (hunk 2 above)
Comment (markrawls) [reply .github/workflows/lint.yml:16]: One would think, but apparently not! There's an issue here:

https://github.com/yarnpkg/yarn/issues/1503

Comment (Ian-MacLeod) [reply .github/workflows/lint.yml:16]: Ah, today I learned!

**Review by Ian-MacLeod — APPROVED** [review]
Lgtm

**Pull request merged**